lib/ansiblelint/__main__.py

```diff
@@ -117,6 +118,12 @@ def main() -> int:
     for match in sorted(set(matches)):
         print(formatter.format(match, options.colored))
 
+    # If run under GitHub Actions we also want to emit output recognized by it.
+    if os.getenv('GITHUB_ACTIONS') == 'true' and os.getenv('GITHUB_WORKFLOW'):
+        formatter = formatters.AnnotationsFormatter(cwd, options.display_relative_path)
```

Comment:
This should always use a relative path. Otherwise, the annotations won't be mapped to things in the project:
```suggestion
        formatter = formatters.AnnotationsFormatter(cwd, True)
```